components/inputs/input-date.js

```diff
@@ -381,7 +393,6 @@ class InputDate extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(LitEl
 		setTimeout(() => {
 			this._textInput.scrollIntoView({ block: 'nearest', behavior: 'smooth', inline: 'nearest' });
 		}, 150);
-		this._dropdownOpened = true;
```

Comment:
Guessing this isn't needed anymore since we're always the ones who open it now?

Reply:
Yup exactly